docs/open-telemetry.asciidoc

```diff
@@ -15,480 +16,71 @@
-      exporters: [logging, otlp/elastic]
-    logs:
-      receivers: [otlp]
-      exporters: [logging, otlp/elastic]
-----
-<1> Environment-specific configuration parameters can be conveniently passed in as environment variables: `ELASTIC_OTLP_ENDPOINT` and `ELASTIC_OTLP_TOKEN`
-
-* Configure the AWS Lambda Java function with:
-** https://docs.aws.amazon.com/lambda/latest/dg/API_Layer.html[Function
-layer]: The latest https://aws-otel.github.io/docs/getting-started/lambda/lambda-java[AWS
-Lambda layer for OpenTelemetry] (e.g. `arn:aws:lambda:eu-west-1:901920570463:layer:aws-otel-java-wrapper-ver-1-2-0:1`)
-** https://docs.aws.amazon.com/lambda/latest/dg/API_TracingConfig.html[`TracingConfig` / Mode] set to `PassTrough`
-** https://docs.aws.amazon.com/lambda/latest/dg/API_FunctionConfiguration.html[`FunctionConfiguration` / Timeout] set to more than 10 seconds to support the longer cold start inherent to the Lambda Java Runtime
-** Export the environment variables:
-*** `AWS_LAMBDA_EXEC_WRAPPER="/opt/otel-proxy-handler"` for wrapping handlers proxied through the API Gateway (see https://aws-otel.github.io/docs/getting-started/lambda/lambda-java#enable-auto-instrumentation-for-your-lambda-function[here])
-*** `OTEL_PROPAGATORS="tracecontext, baggage"` to override the default setting that also enables X-Ray headers causing interferences between OpenTelemetry and X-Ray
-*** `OPENTELEMETRY_COLLECTOR_CONFIG_FILE="/var/task/opentelemetry-collector.yaml"` to specify the path to your OpenTelemetry Collector configuration
-
 [float]
-[[open-telemetry-aws-lambda-java-terraform]]
-===== Instrumenting AWS Lambda Java functions with Terraform
-
-We recommend using an infrastructure as code solution like Terraform or Ansible to manage the configuration of your AWS Lambda functions.
+== OpenTelemetry and the {stack}
 
-Here is an example of AWS Lambda Java function managed with Terraform and the https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function[AWS Provider / Lambda Functions]:
+[subs=attributes+]
+include::./diagrams/apm-otel-architecture.asciidoc[Architecture of Elastic APM with OpenTelemetry]
 
-* Sample Terraform code: https://github.com/cyrille-leclerc/my-serverless-shopping-cart/tree/main/checkout-function/deploy
-* Note that the Terraform code to manage the HTTP API Gateway (https://github.com/cyrille-leclerc/my-serverless-shopping-cart/tree/main/utils/terraform/api-gateway-proxy[here]) is copied from the official OpenTelemetry Lambda sample https://github.com/open-telemetry/opentelemetry-lambda/tree/e72467a085a2a6e57af133032f85ac5b8bbbb8d1/utils[here]
+Elastic integrates with OpenTelemetry, allowing you to reuse your existing instrumentation
+to easily send observability data to the {stack}.
+There are four ways to integrate OpenTelemetry with the {stack}:
 
-[float]
-[[open-telemetry-aws-lambda-nodejs]]
-===== Instrumenting AWS Lambda Node.js functions
+**Elastic OpenTelemetry Bridge**
```

Comment:
Good catch. Agreed that the titles should be consistent.

Are you hoping to move away from the "bridge" terminology? I kinda like the idea from your comment above: `Elastic APM agent OpenTelemetry Bridge`. If you want to move away from "bridge" then `OpenTelemetry API/SDK with Elastic APM agents` makes sense to me. As long as we're consistent and do a good job describing the feature, I think either will work as a title.

Reply:
Related to this thread is the diagram I uploaded in https://github.com/elastic/apm-server/pull/9915#issuecomment-1378186062.

Reply:
The term "bridge" is technically a very good one for what we built in the agents but it might be worth using other terminology because the user doesn't actually care about the bridge. What they care about is that they can use the OTel API and SDK and _somehow_ the data ends up in Elastic. That's why I suggested "OpenTelemetry API/SDK with Elastic APM agents" because it highlights that the user can still use the OTel API/SDK. The term "bridge" might be too vague to the user. What do you think?

Reply:
> What they care about is that they can use the OTel API and SDK and somehow the data ends up in Elastic. 

Got it. ++ on switching then